src/main/java/gregtech/integration/jei/GTJeiPlugin.java

```diff
@@ -172,5 +173,8 @@ public void register(IModRegistry registry) {
         }
 
         registry.addRecipeCatalyst(MetaTileEntities.WORKBENCH.getStackForm(), VanillaRecipeCategoryUid.CRAFTING);
+
+        Arrays.stream(MetaTileEntities.FLUID_CANNER).forEach(machine ->
+            registry.addIngredientInfo(machine.getStackForm(), VanillaTypes.ITEM, "gregtech.machine.fluid_canner.jei_description"));
```

Comment:
Please change this to standard foreach so it is consistent with rest for registrations (for example: https://github.com/GregTechCE/GregTech/blob/master/src/main/java/gregtech/integration/jei/GTJeiPlugin.java#L165)